toptal/picasso
# feat(Slider): add options to hide thumb and track highlight

### Description

Talent Portal uses the `Slider` component to represent survey questions with options ranging from "Strongly Disagree" to "Strongly Agree". The question initially has no selection, but the `Slider` UI in this state makes it look like the first option is selected:

![image](https://user-images.githubusercontent.com/895865/90772275-d26f4480-e2ca-11ea-9871-6706e48c90ed.png)

Another requirement from this same use case is to remove the blue track highlight as it makes it look like the previous options where somehow included:

![image](https://user-images.githubusercontent.com/895865/90773569-7a394200-e2cc-11ea-961c-8576ebbfc49e.png)

### How to test

- Check the new examples on [storybook](https://picasso.toptal.net/tph-231-slider-hide-thumb-track/?path=/story/components-folder--slider) and ensure everything looks fine.
  - Hide thumb when value is null or undefined
  - Disable track highlight

### Screenshots

![image](https://user-images.githubusercontent.com/895865/90774165-64784c80-e2cd-11ea-853d-bcf3660fdd03.png)


### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation


</details>
